articles/connections/database/custom-db/lock.md

```diff
@@ -0,0 +1,11 @@
+---
+description: Learn how to use Lock with your custom database.
+crews: crew-2
+---
+# Using Lock with Custom Databases
```

Comment:
I'm wondering whether this is  doc that we should keep. It doesn't say much so my 2c is that we should either remove it or enhance it.